jenkins-pipelines/longevity-cdc-3d-400gb.jenkinsfile

```diff
@@ -0,0 +1,13 @@
+#!groovy
+
+// trick from https://github.com/jenkinsci/workflow-cps-global-lib-plugin/pull/43
+def lib = library identifier: 'sct@snapshot', retriever: legacySCM(scm)
+
+longevityPipeline(
+    backend: 'aws',
+    aws_region: 'eu-west-1',
+    test_name: 'longevity_test.LongevityTest.test_custom_time',
+    test_config: 'test-cases/longevity/longevity-cdc-3d-400gb.yaml',
+
+    timeout: [time: 360, unit: 'MINUTES']
```

Comment:
The timeout is wrong.

Reply:
yes, i will change, once the stabilize dataset size